ReadingRainbow/MyBooks.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  ContentView.swift
+//  ReadingRainbow
+//
+//  Created by Tyler Sun on 7/20/23.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct MyBooksReducer: Reducer {
+    
+    struct State: Equatable {
+        @PresentationState var myBookDetail: BookDetailReducer.State?
+        var path = StackState<BookDetailReducer.State>()
+    }
+    
+    enum Action: Equatable {
+        case bookDetail(PresentationAction<BookDetailReducer.Action>)
+        case path(StackAction<BookDetailReducer.State, BookDetailReducer.Action>)
+    }
+    
+    var body: some ReducerOf<Self> {
+        Reduce { state, action in
+            switch action {
+                
+            case .bookDetail:
+                return .none
+            case .path:
+                return .none
+            }
+        }
+        .forEach(\.path, action: /Action.path) {
+            BookDetailReducer()
+        }
+        
+    }
+}
+
+
+
+struct MyBooks: View {
+    let store: StoreOf<MyBooksReducer>
+
+    let columns = [GridItem(.adaptive(minimum: 150))]
+ 
+    var body: some View {
+        NavigationStackStore(self.store.scope(state: \.path, action: { .path($0) })) {
+            WithViewStore(store, observe: { $0 }) { ViewStore in
+                ScrollView {
+                    LazyVGrid(columns: columns, spacing: 20) {
+                        ForEach(1..<11) { imageName in
+                            VStack {
+                                Text("name")
+                                
+                                NavigationLink(state: BookDetailReducer.State(bookName: "Ocean\(imageName)")) {
+                                    Image("Ocean\(imageName)").resizable().frame(width: 150, height: 150)
```

Comment:
lets remove the testing text here. If we need testing data then I'd suggest making a Dependency that we can pass in and override with the fake data when needed. 
Maybe that's thing we override is the imageCache object that takes a URL and returns an image. The fake one can just return a fake image randomly or something rather than making a network call. 
But that's probably out of the scope of this ticket.